### DISCUSSION TITLE
Curser being locked into one half of the screen

### DISCUSSION THREAD

**Discussion by FoxxClawz** [ppy/osu#24630] [Q&A]
### Type

Crash to desktop

### Bug description

I am using a Gaomon tablet, and when i use it with lazer it runs into an issue where my curser gets locked into one half of the screen. when i play on stable this issue does not occur. on stable i use war input with no window restriction. I also play with two displays, but my gaomon work area is specified to the right monitor. lazer does not allow me to use the same settings i do in stable (I.e. no window restriction & increased sensitivity from raw input) btw i am using a couple custom rulesets that ended up being phased out by recent updates, but they shouldn't have any contribution to this particular issue.

### Version

Lazer 2023.815.0

### Logs

[input.log](https://github.com/ppy/osu/files/12417719/input.log)
[network.log](https://github.com/ppy/osu/files/12417720/network.log)
[performance.log](https://github.com/ppy/osu/files/12417721/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12417722/runtime.log)
[updater.log](https://github.com/ppy/osu/files/12417723/updater.log)
[database.log](https://github.com/ppy/osu/files/12417724/database.log)

**Comment by FoxxClawz** [comment]
BTW this is my first report so please tell me if i need to provide any other information, or if there is already a fix. I went looking already but i didn't find any fixes for my particular issue.

**Comment by peppy** [comment]
Either turn off raw input or use the built-in tablet support.

**Reply by slopslug** [reply]
Are there going to be any patches to allow us to use third party installations of OpenTabletDriver? I appreciate the thought of putting first party support into lazer, however without any particular way to migrate my configuration or the ability to use custom filters I can't see myself using the built-in version.
